$MirOS: ports/devel/pango/patches/patch-pango_pc_in,v 1.2 2005/09/22 19:54:57 bsiegert Exp $
--- pango.pc.in.orig	2009-02-07 00:49:34.000000000 +0100
+++ pango.pc.in	2009-07-17 17:35:34.000000000 +0200
@@ -9,5 +9,5 @@ Name: Pango
 Description: Internationalized text handling
 Version: @VERSION@
 Requires: glib-2.0,gobject-2.0,gmodule-no-export-2.0
-Libs: -L${libdir} -lpango-@PANGO_API_VERSION@ @PKGCONFIG_MATH_LIBS@
+Libs: -L${libdir} @PKGCONFIG_X_LIBS@ -lpango-@PANGO_API_VERSION@ @PKGCONFIG_MATH_LIBS@
 Cflags: -I${includedir}/pango-1.0
